src/views/ModelPlan/TaskList/Basics/index.tsx

```diff
@@ -1,29 +1,439 @@
+              const {
+                dirty,
+                errors,
+                handleSubmit,
+                setErrors,
+                setFieldValue,
+                isValid,
+                values
+              } = formikProps;
+              const flatErrors = flattenErrors(errors);
+              return (
+                <>
+                  {Object.keys(errors).length > 0 && (
+                    <ErrorAlert
+                      testId="formik-validation-errors"
+                      classNames="margin-top-3"
+                      heading="Please check and fix the following"
+                    >
+                      {Object.keys(flatErrors).map(key => {
+                        return (
+                          <ErrorAlertMessage
+                            key={`Error.${key}`}
+                            errorKey={key}
+                            message={flatErrors[key]}
+                          />
+                        );
+                      })}
+                    </ErrorAlert>
+                  )}
+                  <Form
+                    onSubmit={e => {
+                      handleSubmit(e);
+                      window.scrollTo(0, 0);
+                    }}
+                  >
+                    <FieldGroup
+                      scrollElement="modelName"
+                      error={!!flatErrors.modelName}
+                      className="margin-top-4"
+                    >
+                      <Label htmlFor="plan-basics-model-name">
+                        {t('modelName')}
+                      </Label>
+                      <FieldErrorMsg>{flatErrors.modelName}</FieldErrorMsg>
+                      <Field
+                        as={TextInput}
+                        error={!!flatErrors.modelName}
+                        id="plan-basics-model-name"
+                        maxLength={50}
+                        name="modelName"
+                        defaultValue={modelName}
+                      />
+                    </FieldGroup>
+
+                    <FieldGroup
+                      scrollElement="modelCategory"
+                      error={!!flatErrors.modelCategory}
+                      className="margin-top-4"
+                    >
+                      <Label htmlFor="plan-basics-model-category">
+                        {t('modelCategory')}
+                      </Label>
+                      <FieldErrorMsg>{flatErrors.modelCategory}</FieldErrorMsg>
+                      <Field
+                        as={Dropdown}
+                        id="plan-basics-model-category"
+                        name="role"
+                        value={values.modelCategory}
+                        onChange={(e: any) => {
+                          setFieldValue('modelCategory', e.target.value);
+                        }}
+                      >
+                        <option value="" key="default-select" disabled>
```

Comment:
Here you can remove the value prop of the default option and instead add the prop `selected` to have the null value be the default while still displaying the `-Select-` string